osquery/tables/system/linux/rpm_packages.cpp

```diff
@@ -150,6 +150,7 @@ QueryData genRpmPackages(QueryContext& context) {
     r["size"] = getRpmAttribute(header, RPMTAG_SIZE, td);
     r["sha1"] = getRpmAttribute(header, RPMTAG_SHA1HEADER, td);
     r["arch"] = getRpmAttribute(header, RPMTAG_ARCH, td);
+    r["epoch"] = getRpmAttribute(header, RPM_EPOCH, td);
```

Comment:
Looks like you've caught some other code changes up in this? Perhaps try rebasing first before we move forward to ensure you're only modifying the certificates table.